src/components/sbb-slider/sbb-slider.stories.tsx

```diff
@@ -16,7 +19,7 @@ const TemplateSbbSlider = (args): JSX.Element => <sbb-slider {...args}></sbb-sli
 const TemplateSbbSliderChangeEvent = (args): JSX.Element => (
   <Fragment>
     <sbb-slider {...args} onChange={(event) => changeEventHandler(event)}></sbb-slider>
-    <div style={{ 'margin-block-start': '2rem' }}>Change slider position:</div>
+    <div style="margin-block-start: 2rem;">Change slider position:</div>
```

Comment:
Why is this change needed?

Reply:
It's just before this would have been a type error or something (but at the moment deactivated), and as we did not change it else, it's maybe better to keep the old notation until we migrate stories anyways